packages/amplication-design-system/src/style/css-variables.scss

```diff
@@ -54,20 +54,30 @@
 
   --form-horizontal-width: 700px;
   --form-horizontal-label-width: 300px;
-  --title1-font-size: 15px;
+  --large-icon-font-size: 18px;
+  --title1-font-size: 16px;
```

Comment:
did you check where else we use this var and what is the impact?

Reply:
I just added a var and used it. What can it affect?

Reply:
I thought1 this is not a new one var

Reply:
`--title1-font-size` => always been there
`--large-icon-font-size` => the variable I added